### PR TITLE
Changes to paths and imports

### PR DESCRIPTION
**Pull request by nitech**
- loading-indicator.ts/.js is in the same folder as index.ts
- bootstrap and fontawesome-imports were missing in the last example of app.ts

### CODE REVIEW

File: current/en-us/2. tutorials/2. creating-a-contact-manager.md [hunk 1]
```diff
@@ -754,14 +754,14 @@ Previously, when we created the `contact-list` component, we required that into
 import {PLATFORM} from 'aurelia-framework';
 
 export function configure(config) {
-  config.globalResources([PLATFORM.moduleName('./elements/loading-indicator')]);
+  config.globalResources([PLATFORM.moduleName('./loading-indicator')]);
```
Comment (EisenbergEffect) [inline current/en-us/2. tutorials/2. creating-a-contact-manager.md]: @bigopon Is this a possible difference between Webpack and Require.js or will it work the same in both cases?

Comment (nitech) [reply current/en-us/2. tutorials/2. creating-a-contact-manager.md]: Note that I tested this with TypeScript - not with ES

**Comment by 3cp** [comment]
@nitech in the tutorial, it said

> In the src/resources/elements folder create a file named loading-indicator.ts

The path was correctly reflecting the location.

**Comment by 3cp** [comment]
Also bootstrap and fontawesome were not missing from app.ts, the tutorial covered those import in `main.ts`

> Next, we need to import it in our main.ts by adding the following line to the top of the file:
import 'bootstrap/dist/css/bootstrap.css';
import 'font-awesome/css/font-awesome.css';

**Comment by nitech** [comment]
> @nitech in the tutorial, it said
> 
> > In the src/resources/elements folder create a file named loading-indicator.ts
> 
> The path was correctly reflecting the location.

Aw, you are so right. Sorry for wasting your time @3cp and @EisenbergEffect. Not my intention! 

Great tutorial though. 👍 to the team.

**Comment by 3cp** [comment]
No worry :-)
We all do jump reading, especially on tech doc. I blame computer and info tech destroyed my ability to stay focus.

**Comment by EisenbergEffect** [comment]
Definitely nothing to worry about @nitech We're always glad to have community members that care enough to help us improve things. If you come across anything else that may need fixing or clarification, please don't hesitate to send a PR 😄

**Pull request closed without merging**